automatic/vlc/vlc.nuspec

```
@@ -41,4 +41,7 @@ VLC is a free and open source cross-platform multimedia player and framework tha
       <dependency id="vlc.install" version="[3.0.21]" />
     </dependencies>
   </metadata>
+  <files>
+    <file src="vlc.nuspec" />
+  </files>
```

This is the wrong way to prevent unnecessary files from being embedded.
A nuspec file is always embedded, so this should not be part of the `files` section (unless the intention had been to embed a different nuspec file in addition).
As such, this section should just be:

```suggestion
  <files />
```